packages/truffle-core/cli.js

```diff
@@ -33,15 +33,7 @@ const Command = require("./lib/command");
 
 const command = new Command(require("./lib/commands"));
 
-// Hack to suppress web3 MaxListenersExceededWarning
-// This should be removed when issue is resolved upstream:
-// https://github.com/ethereum/web3.js/issues/1648
-const listeners = process.listeners("warning");
-listeners.forEach(listener => process.removeListener("warning", listener));
```

Comment:
Are you sure we can remove this?

Reply:
Where is the output for this supposed to occur?

Reply:
Maybe I'm reading Coveralls incorrectly, but it looks like on `develop` these suppression lines are still getting hit.

Reply:
I removed it and I haven't seen any adverse effects yet. I guess we should test everything pretty well to see if it is just leftover junk or not :)

Reply:
Following the trail of issues/PRs...

https://github.com/ethereum/web3.js/issues/1648
https://github.com/trufflesuite/ganache-core/issues/267
https://github.com/trufflesuite/ganache-core/pull/269

It looks like this hasn't been resolved yet in the version of web3 we're using.
Also looks like the `ganache-core` PR would help when using Ganache as a provider, but there's still web3-specific event emitting issues.